src/dotnet/commands/dotnet-new/Program.cs

```diff
@@ -64,11 +65,37 @@ where resourceName.Contains(templateDir)
                             }
                         }
 
-                        archive.ExtractToDirectory(Directory.GetCurrentDirectory());
+                        string projectDirectory = Directory.GetCurrentDirectory();
+
+                        archive.ExtractToDirectory(projectDirectory);
+
+                        string projectJsonFile = Path.Combine(projectDirectory, "project.json");
```

Comment:
Good point.  I should have caught this myself.

---
In reply to: [71796369](https://github.com/dotnet/cli/pull/3911#discussion_r71796369) [](ancestors = 71796369)